src/remote-config/remote-config-api.ts

```diff
@@ -399,6 +399,12 @@ export interface ServerTemplate {
    * project's {@link ServerTemplate}.
    */
   load(): Promise<void>;
+
+  /** 
+   * Convenient method that returns the JSON string of the cached template data
+   * @returns A JSON-string of this object.
```

Comment:
Nit: I think we can drop the "Convenient method that" since this is now the only way to get template data.

Reply:
Discussed offline, we're going to remove this method for now pending amendment 4 in the proposal doc